### PR TITLE
[TOOLS-200] Refactor Secrets & update pterm style

### PR DESCRIPTION
**Pull request by AutomationD**
# Testing Done
## Default File path
[asciicast](https://asciinema.org/a/PrK6gCDf8TjXtySKS0QQDih8P)

## Custom File path
(I forgot the --file parameter name first :-D)
[asciicast](https://asciinema.org/a/dOBcU3qHKYDGfPTlP05ALIJq4)

[TOOLS-200]

[TOOLS-200]: https://hazelops.atlassian.net/browse/TOOLS-200?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by psihachina** [comment]
@AutomationD I don't see the test)

**Comment by AutomationD** [comment]
> @AutomationD I don't see the test)

Tests uploaded.

**Comment by psihachina** [comment]
> Tests uploaded.

Your test looks beautiful. There is something to learn. These tests can be used to write examples for this command :)

**Review by psihachina — CHANGES_REQUESTED** [review]

File: cmd/main.go [hunk 1]
```diff
@@ -23,8 +23,30 @@ import (
...
+	pterm.Success.Prefix = pterm.Prefix{
+		Text:  "✓",
+		Style: pterm.NewStyle(pterm.FgGreen),
+	}
+
+	pterm.Error.Prefix = pterm.Prefix{
+		Text:  "✗",
+		Style: pterm.NewStyle(pterm.FgRed),
+	}
+
```
[the remark this answers is not in the document]
Comment (AutomationD) [reply cmd/main.go:43]: Let's keep it in mind, whenever you know the place, feel free to move.

File: internal/commands/secrets/secrets_push.go [hunk 2]
```diff
@@ -16,29 +11,39 @@ import (
...
-	Region  string
-	Profile string
-	Type    string
-	Path    string
-	Force   bool
+type SecretsPushOptions struct {
+	Env      string
+	Region   string
+	Profile  string
+	AppName     string
```
[the remark this answers is not in the document]
Comment (AutomationD) [reply internal/commands/secrets/secrets_push.go:23]: @psihachina We also use it in Run. I'd keep it like that, any objections?

**Review by psihachina — APPROVED** [review]
LGTM

**Pull request merged**